CoreData/NSManagedObject+S2MAdditions.m

```diff
@@ -0,0 +1,306 @@
+//
+//  NSManagedObject+S2MAdditions.m
+//  S2MToolbox
+//
+//  Created by ParkSanggeon on 13. 10. 22..
+//  Copyright (c) 2013년 SinnerSchrader Mobile. All rights reserved.
+//
+
+#import "NSManagedObject+S2MAdditions.h"
+
+@implementation NSManagedObject (S2MAdditions)
+
++ (NSEntityDescription*)entityInManagedObjectContext:(NSManagedObjectContext*)moc_
+{
+#warning use mogenerator
```

Comment:
Agreed, it seems to be more a dependency, we  could remove the method and move the declaration to a protocol. Then use custom mogenerator template to declare the method